ledger-core/v2/application/functest/wallet_get_balance_test.go

```diff
@@ -31,13 +29,11 @@ func TestWalletGetBalance(t *testing.T) {
 
 	require.Empty(t, resp.Err, "problem during execute request")
 	assert.NotEmpty(t, resp.TraceID, "traceID mustn't be empty")
-	assert.Equal(t, 1000, resp.Amount, "wrong amount")
+	assert.Equal(t, 1000000000, resp.Amount, "wrong amount")
```

Comment:
I think it's better to use the same ** initialBalance** const here

Reply:
nope, tests shouldn't use constants from code, they can use a constant, but defined in tests